client/extensions/woocommerce/app/product-categories/index.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionHeader from 'woocommerce/components/action-header';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import Main from 'components/main';
+import NavTabs from 'components/section-nav/tabs';
+import NavItem from 'components/section-nav/item';
+import SectionNav from 'components/section-nav';
+import Search from 'components/search';
+
+class ProductCategories extends Component {
+
+	render() {
+		const { className, translate, site } = this.props;
+		const classes = classNames( 'product_categories__list', className );
+
+		const productsLabel = translate( 'Products' );
+		const categoriesLabel = translate( 'Categories' );
+
+		return (
+			<Main className={ classes } wideLayout>
+				<ActionHeader breadcrumbs={ [
+					<a href={ getLink( '/store/products/:site/', site ) }>{ productsLabel }</a>,
+					<span>{ categoriesLabel }</span>,
+				] }>
+				</ActionHeader>
+				<SectionNav>
+					<NavTabs label={ translate( 'Products' ) } selectedText={ categoriesLabel }>
+						<NavItem path={ getLink( '/store/products/:site/', site ) }>{ productsLabel }</NavItem>
+						<NavItem path={ getLink( '/store/products/categories/:site/', site ) } selected>
+							{ categoriesLabel }
+						</NavItem>
+					</NavTabs>
+
+					<Search
+						pinned
+						fitsContainer
+						placeholder={ translate( 'Search categories…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **80** times:
`translate( 'Search categories' )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cats-base)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).